doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1109,21 +1109,21 @@ Zephyr
 
 .. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
 
-The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``a8b28f13c195a00bdf50f5c24092981124664ed9``, with some |NCS| specific additions.
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``a768a05e6205e415564226543cee67559d15b736``, with some |NCS| specific additions.
 
 For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
 
 .. code-block:: none
 
-   git log --oneline a8b28f13c1 ^4bbd91a908
+   git log --oneline a768a05e62 ^4bbd91a908
```

Comment:
Eh, because I confused NCS version with NCS tag-on-upstream, and checked where the 2.4 line starts,  rather than the tag for upstream version we have taken in.